src/pupil_labs/invisible_lsl_relay/cli.py

```diff
@@ -86,7 +89,10 @@ async def get_device_identifier(device_ip, device_port):
                 'is connected to the same network.'
             )
             raise exc
-        return status.phone.device_id
+        assert (
+            status.hardware.world_camera_serial
+        ), "The world camera is not connected to the device."
```

Comment:
I feel like this should rather be a warning than an assertion. Let's fall back to `"default"`. This way, if one was to request the intrinsics for this acquisition field, one would still get a [valid response](https://api.cloud.pupil-labs.com/v2/hardware/default/calibration.v1?json). This also corresponds to the intrinsics used by the Companion app if no scene camera is connected.

Reply:
If the camera field is none, LSL cannot write it into a string and crashes later. But I can return and empty string if the camera is not connected
.

Reply:
I was suggesting to fallback to the literal string `"default"` if the scene camera is not connected.
```suggestion
        serial = status.hardware.world_camera_serial or "default"
        return status.phone.device_id, serial
```

Reply:
Ah! Sorry, I didn't get that. Sounds good!